src/components/directory-breadcrumbs.js

```diff
@@ -66,6 +67,11 @@ const DirectoryBreadcrumbs = () => {
     const selectedDirectory = useSelector((state) => state.selectedDirectory);
     const currentPath = useSelector((state) => state.currentPath);
 
+    function limitChar(str, limit) {
+        return str.length > limit ? `${str.slice(0, limit)}...` : str;
+    }
+    const limitElementNameLength = 160;
```

Comment:
Defining it before limitChar (mostly for clarity), you could have used it directly in limitChar, avoiding repeating it in the calls.

Reply:
it's fixed